src/Explorer/SplashScreen/SplashScreen.tsx

```diff
@@ -574,9 +574,9 @@ export class SplashScreen extends React.Component<SplashScreenProps> {
             description: "Learn about the Azure Cosmos DB SDK.",
           },
           {
-            link: "https://aka.ms/msl-complex-queries",
-            title: "Master Complex Queries",
-            description: "Learn how to author complex queries.",
+            link: "https://developer.azurecosmosdb.com/tv",
```

Comment:
If we're making them all to be same, let's make a variable instead. That way if a minor change comes in (like switching to an aka.ms link) the next person only has to change one line.

Reply:
Updating to make a cdbLiveTv const.